javav2/example_code/lookoutvision/Readme.md

```diff
@@ -22,19 +58,27 @@ You can execute the JUnit tests from a Java IDE, such as IntelliJ, or from the c
 
 	Test 3 passed
 
-**WARNING**: _Running these JUnit tests manipulates real AWS resources and might incur charges on your account._
+**WARNING**: _Running these JUnit tests manipulates real Amazon Lookout for Vision models. You are charged for the amount of time it takes to successfully train the test model and for the amount of time the test model is hosted._
 
  ### Properties file
-Before running the Amazon Lookout for Vision JUnit tests, you must define values in the **config.properties** file located in the **resources** folder. This file contains values that are required to run the JUnit tests. For example, you define an instance name used for various tests. If you do not define all values, the JUnit tests fail.
+Before running the Amazon Lookout for Vision JUnit tests, you must define values in the **config.properties** file located in the **resources** folder. This file contains values that are required to execute the JUnit tests. For example, you define an instance name used for various tests. If you do not define all values, the JUnit tests fail.
 
 Define these values to successfully run the JUnit tests:
 
-- **projectName** - The name of the Lookout for Vision project that you want to use.   
-- **modelVersion** – The version of the model within the project that you want to use.  
-- **photo** - The path to an image that you want to analyze.
 
+- **projectName** -  The name of the project to create in the tests.
+- **modelDescription** - A description for the model.
+- **modelTrainingOutputBucket** - The Amazon S3 bucket in which to place the training results.
+- **modelTrainingOutputFolder** - The folder in modelTrainingOutputBucket in which to place the training results.
+- **photo** - The location of an image to analyze with the trained model. 
+- **manifestFile** - The location of a local manifest file that is used to populate the training dataset. For more information, see [Creating a manifest file](https://docs.aws.amazon.com/lookout-for-vision/latest/developer-guide/manifest-files.html). 
+- **modelPackageJobJsonFile** -  The location of the edge packaging Job request JSON file. We provide a template JSON file for a [target device](./src/main/resources/packaging-job-request-device-template.json) (Jetson Xavier) and a template JSON file for a [target platform](./src/main/resources/packaging-job-request-hardware-template.json). To successfully run the model packaging job test, make sure the value of **ModelVersion** is "1". Each time you run the test, you must change the value of **ComponentVersion** and **JobName**. For information about the package settings that you can make, see [Packaging your model (SDK)](https://docs.aws.amazon.com/lookout-for-vision/latest/developer-guide/package-job-sdk.html).
+
+If you want to use the project, dataset, and model and that testing creates, disable the following tests:
```

Comment:
and model and that testing creates, --> and model that the testing creates,

Reply:
Fixed.